components/raftstore-v2/src/operation/life.rs

```diff
@@ -89,6 +90,43 @@ impl DestroyProgress {
 }
 
 impl Store {
+    /// The method is called during split.
+    /// The creation process is:
+    /// 1. create an uninitialized peer if not existed before
+    /// 2. initialize the peer by the information sent from parent peer
+    #[inline]
+    pub fn on_peer_creation<EK, ER, T>(
+        &mut self,
+        ctx: &mut StoreContext<EK, ER, T>,
+        msg: Box<CreatePeer>,
+    ) where
+        EK: KvEngine,
+        ER: RaftEngine,
+    {
+        let region_id = msg.as_ref().region.id;
+        let mut raft_msg = RaftMessage::default();
+        raft_msg.set_region_id(region_id);
+        raft_msg.set_region_epoch(msg.as_ref().region.get_region_epoch().clone());
+        raft_msg.mut_from_peer().set_id(raft::INVALID_ID);
+        raft_msg.set_to_peer(
+            msg.as_ref()
+                .region
+                .get_peers()
+                .iter()
+                .find(|p| p.get_store_id() == ctx.store_id)
```

Comment:
store_id can be queried by `self.store.id`.

Reply:
okay